### PR TITLE
Show angle of camera

### LINKED ISSUE #633: Show angle of camera
The camera angle would help us set angle of Lara properly to get ALTs - useful for researching.

### PR DESCRIPTION
**Pull request by chreden**
Show the angle of the camera above the camera position display.
Add a setting for whether to show the angle in degrees or radians - by default this is set to degrees because most people will probably want that.
Add tests for most of what was changed, created issues where it would be too much change.
Settings window gets an interface and some tests.
Closes #633

### CODE REVIEW

File: trview.app/Settings/SettingsLoader.cpp [hunk 1]
```diff
@@ -57,6 +57,7 @@ namespace trview
             read_attribute(json, settings.rooms_startup, "roomsstartup");
             read_attribute(json, settings.camera_acceleration, "cameraacceleration");
             read_attribute(json, settings.camera_acceleration_rate, "cameraaccelerationrate");
+            read_attribute(json, settings.camera_display_degrees, "cameradisplaydegrees");
```
Comment (chreden) [inline trview.app/Settings/SettingsLoader.cpp:60]: This needs tests

File: trview.app/Windows/Viewer.cpp [hunk 2]
```diff
@@ -127,6 +127,11 @@ namespace trview
             }
             _free_camera.set_position(position);
         };
+        _token_store += _ui->on_camera_rotation += [&](auto yaw, auto pitch)
+        {
+            current_camera().set_rotation_yaw(yaw);
```
Comment (chreden) [inline trview.app/Windows/Viewer.cpp:132]: Needs test

File: trview.app/UI/ViewerUI.cpp [hunk 3]
```diff
@@ -163,9 +163,15 @@ namespace trview
             _settings.camera_acceleration_rate = value;
             on_settings(_settings);
         };
+        _token_store += _settings_window->on_camera_display_degrees += [&](bool value)
+        {
```
Comment (chreden) [inline trview.app/UI/ViewerUI.cpp:167]: Tests

File: trview.app/Settings/SettingsLoader.cpp (hunk 1 above)
Comment (chreden) [reply trview.app/Settings/SettingsLoader.cpp:60]: Created #810 to deal with this.

**Review by makotocchi — APPROVED** [review]

**Pull request merged**